src/fsspec_xrootd/xrootd.py

```diff
@@ -18,13 +21,28 @@ class XRootDFileSystem(AbstractFileSystem):  # type: ignore[misc]
     protocol = "root"
     root_marker = "/"
 
-    def __init__(self, *args: list[Any], **storage_options: str) -> None:
+    def __init__(self, *args: list[Any], **storage_options: Any) -> None:
+        self.timeout = storage_options.get("timeout", 0)
         self._path = storage_options["path"]
         self._myclient = client.FileSystem(
             storage_options["protocol"] + "://" + storage_options["hostid"]
         )
+        status, _n = self._myclient.ping(self.timeout)
```

Comment:
Probably the ping should have a hardcoded shorter timeout (15s is apparently the smallest without changing an environment variable) to detect misspelled servers quickly

Reply:
How do I do that?